quickevent/app/quickevent/plugins/Relays/src/relaysplugin.cpp

```diff
@@ -289,7 +289,7 @@ qf::core::utils::TreeTable RelaysPlugin::nLegsClassResultsTable(int class_id, in
 				.joinRestricted("runs.relayId", "relays.id", "relays.classId=" QF_IARG(class_id), qfs::QueryBuilder::INNER_JOIN)
 				//.where("runs.relayId IN (" + relay_ids.join(',') + ")")
 				.where("runs.leg>0 AND runs.leg<=" + QString::number(leg_count))
-				.orderBy("runs.relayId, runs.leg");
+				.orderBy("runs.relayId, runs.leg, runs.disqualified, runs.timeMs");
```

Comment:
Order by disqualified in DESC order (so that valid punching appears first)?

Reply:
`disqualified` is bool type, booleans in sql are ordered as false before true.